ocean_lib/common/ddo/service.py

```diff
@@ -128,7 +134,9 @@ def as_dictionary(self):
         return values
 
     @classmethod
-    def _parse_json(cls, service_dict):
+    def _parse_json(
+        cls, service_dict: Dict[str, Any]
+    ) -> Tuple[str, str, int, Dict, Dict]:
```

Comment:
After adding `@enforce_types` does specifying `Dict` like this without specifying the contained types actually work?

Reply:
yes, I added enforce_types everywhere (possible) as of now.